kernelci/db/kernelci_api.py

```diff
@@ -91,7 +91,7 @@ def get_node(self, node_id):
     def get_nodes(self, attributes: dict = None):
         """Get all nodes matching attributes"""
         resp = self._get('nodes', params=attributes)
-        return resp.json()
+        return (resp.json()).get('items')
```

Comment:
```suggestion
        return resp.json().get('items')
```
In fact, could we just use `['items']` instead?  The API has the return type set to `PageModel` so it should be something we can rely on.